src/api/app/models/staging/staging_project.rb

```diff
@@ -59,6 +59,17 @@ def overall_state
       @overall_state ||= state
     end
 
+    def assign_managers_group(managers)
```

Comment:
I think we should do it in a callback. This way we don't need to do it always manually. We could check if the manager of the workflow changed and if yes we update the assocation.

Sth like
```ruby
before_save :update_staging_managers

def update_staging_managers
       managers = staging.staging_workflow.managers
       role = Role.find_by_title!('maintainer')
       return if relationships.find_by(group: managers, role: role)
       Relationship.add_group(self, managers, role, nil, true)
end
```

(untested)